apps/web/app/routes/stacking/liquid-stacking.tsx

```diff
@@ -0,0 +1,41 @@
+import { StackingClientProvider } from '~/features/stacking/providers/stacking-client-provider';
+import { validateProtocolSlug } from '~/features/stacking/utils/stacking-liquid-protocols-validator';
+import {
+  ProtocolIdToDisplayNameMap,
+  ProtocolSlug,
+  ProtocolSlugToIdMap,
+} from '~/features/stacking/utils/types-preset-protocols';
+import { LiquidStacking } from '~/pages/stacking/liquid-stacking';
+
+import { delay } from '@leather.io/utils';
+
+import { Route } from './+types/liquid-stacking';
+
+export async function loader({ params }: Route.LoaderArgs) {
+  await delay(400);
+
+  const isSlugValid = validateProtocolSlug(params.slug);
+
+  if (!isSlugValid) {
+    throw new Response(`Invalid protocol slug: ${params.slug}`, { status: 404 });
+  }
+
+  const protocolSlug = params.slug as ProtocolSlug;
```

Comment:
use zod string validator https://trustmachines.slack.com/archives/C05LAP952E7/p1743775904159909

Reply:
added protocolSlugSchema, though not sure how to avoid type cast in it